docs/variants/dell_optiplex/initial-deployment.md

```diff
@@ -67,33 +67,22 @@ active. Press F1 to proceed and boot to your Linux system.
 
     ![](../../images/service_mode_warn.jpg)
 
-## Initiali deployment
+## Initial deployment
 
-For simplicity, we are using Dasharo Tools Suite SE version booted from USB.
-Access to DTS SE is available for supporters using [multiple
+For simplicity, we are using Dasharo Tools Suite SE version 1.0.3 booted from
+USB. Access to DTS SE is available for supporters using [multiple
 methods](../../osf-trivia-list/dts/#how-to-get-dasharo-tools-suite-supporters-edition).
```

Comment:
Probably, this is not the scope of this task, but this link is empty. Do we accept broken links in published documentation?

Reply:
@sulewskiprzemyslaw thank you, fixed https://github.com/Dasharo/docs/pull/352/commits/720abea65ecc9b8ec5b7c4d47315055feae52431#diff-790a9c00e523ff125b88a6094447eb5c3c57a701b736430969c4b6849fa034a1R74